src/theme/src/ThemeContext.ts

```diff
@@ -49,6 +49,7 @@ export interface Theme {
   getLinkClassName: (color?: string) => string
   getParagraphStyle: (size?: number) => any
   getTagInputClassName: (alias: 'default') => string
+  getTextareaClassName: typeof defaultTheme.getTextareaClassName
```

Comment:
I think this is probably fine, but any reason to deviate from the others here?

Reply:
Should we adopt this style for the rest of these functions on the theme?

Reply:
I was pre-empting #577, expecting it to be merged first, where @Rowno adopts this style across all these functions: https://github.com/segmentio/evergreen/pull/577/files/51ddc1b740c211224cdc1ebe736fde77657d3395#diff-06dbe722e90e578810dc6405c11a6f96

Reply:
Makes sense! Sounds good to me.